ui/app/components/auth-jwt.js

```diff
@@ -68,7 +68,9 @@ export default Component.extend({
       }
     }
     this.set('role', role);
-  }).restartable(),
+  })
+    .restartable()
+    .withTestWaiter(),
```

Comment:
Should we remove this since we're skipping auth-jwt tests in IE?

Reply:
i thought about leaving it in to prevent concurrency flakiness in the chrome tests, but since it doesn't seem to be a real issue i can definitely take it out.